forecasting_sa/Forecaster.py

```diff
@@ -322,11 +321,11 @@ def objective(params, train_df_path, val_df_path):
             )
             # TODO final train should be configurable
             # check if we should use all the data for final retrain
-            best_model.fit(train_df)
+            best_model.fit(train_df.append(val_df))
```

Comment:
I'm open to discussions, but I do believe that the best model has to be refitted using the entire dataset.